doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc

```diff
@@ -0,0 +1,34 @@
+[id='test-designer-create-test-template-dmn-based-proc']
+= Creating a test scenario template for DMN-based test scenarios
+
+Create a test scenario template for DMN-based scenarios by following the procedure below to validate your rules and data.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project for which you want to create the test scenario.
+. Click *Add Asset* -> *Test Scenario*.
+. Enter a *Test Scenario* name and select the appropriate *Package*. The package you select must contain all the required data objects and rule assets have been assigned or will be assigned.
```

Comment:
Minor suggestion - shorten sentence and simplify:

. Enter a *Test Scenario* name and select the appropriate *Package*. The package you select must contain all the required data objects and rule assets have been assigned or will be assigned./. Enter a *Test Scenario* name and select the appropriate *Package*. The package you select must contain all the required data objects and rule assets assigned.

Reply:
Changed as suggested.